clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2606,13 +2637,25 @@ private void pollStatusUntilComplete(
       });
 
       if (overallStatus.isTerminal()) {
+        if (pushJobSetting.canaryRegionPush) {
+          if (completedDatacenters.size() == 1) {
+            LOGGER.info("Successfully pushed {} to {}", topicInfo.topic, completedDatacenters.iterator().next());
+            return;
+          } else {
+            throw new VeniceException(
+                "Canary region push job ended up with zero or more than one region ending with unexpected COMPLETED status, "
```

Comment:
it cannot be more than 1 right ? may be less than 1. accordingly update the message.

Reply:
I'm being defensive here in case the canary region push feature is broken for whatever reasons and 0 or more than 1 regions ingest the data.